src/wallet-adapters/ledger.ts

```diff
@@ -0,0 +1,92 @@
+// modified from https://github.com/project-serum/spl-token-wallet/blob/master/src/utils/walletProvider/ledger-core.js
+// @TODO: check license for ^
```

Comment:
you can remove this, the code was originally from the Solana repo which is Apache.